### DISCUSSION TITLE
Can outerFollowSet be private? Asking because it isn't reaped even when a parser isn't using it at all ...

### DISCUSSION THREAD

**Discussion by vsajip** [congo-cc/congo-parser-generator#132] [Q&A] [answered]
https://github.com/congo-cc/congo-parser-generator/blob/73c94d77cedf33a28da57d34a9ab4ee1d88ad8e9/src/templates/java/Parser.java.ftl#L68

**Comment by revusky** [comment, marked as answer]
It certainly looks like it can perfectly well be private. I don't know why it isn't. Maybe there was some earlier implementation where it needed to have package visibility.
